js/ui/status/automaticUpdates.js

```diff
@@ -221,8 +221,7 @@ var Indicator = class extends PanelMenu.SystemIndicator {
 
     _updateNotification() {
         // Only notify when in an regular session, not in GDM or initial-setup.
-        if (Main.sessionMode.currentMode != 'user' &&
-            Main.sessionMode.currentMode != 'user-coding') {
```

Comment:
It's a bit unfortunate that the `user-coding` bit leaked through the 3.32 rebase, as it is a leftover from the previous version of flip to hack. When we next rebase, we should make sure not to have this here in the first place.
Could you split this change out into its own commit, and mention in the commit message that it should be squashed with b7bb99e4ae7f5dde4dd3a47d30d2c322cecb52df upon the next rebase?

(Tagging @GeorgesStavracas for posterity.)